spec/rspec/core/world_spec.rb

```diff
@@ -22,6 +22,30 @@ module RSpec::Core
       end
     end
 
+    describe "#examples_count" do
+      before do
+        configuration.filter_run_including :foo => "bar"
+      end
+
+      it "returns examples count that includes shared examples" do
+        RSpec.describe "eg1" do
+          shared_context "sc1", :foo do
+          end
```

Comment:
I don't understand what this shared context has to do with this spec.  Can you explain?